google-cloud-spanner/acceptance/spanner_helper.rb

```diff
@@ -35,6 +35,23 @@ def emulator_enabled?
   ENV["SPANNER_EMULATOR_HOST"]
 end
 
+def cleanup_all_databases(instance)
+  return if emulator_enabled? || instance.nil?
+
+  puts "Cleaning up all databases."
+
+  instance.databases.all do |database|
+    next unless database.database_id.start_with?('gcruby')
```

Comment:
Please also only delete stale databases that were created 1 day ago. I am worried that if multiple tests running at the same time will affect each other.